### PR TITLE
Add non .ini wpt meta handler

### LINKED ISSUE #43: Warn if non-ini files are added to tests/wpt/metadata/ or tests/wpt/mozilla/meta/
This would entail a new handler which looks at the new PR's diff (via `get_diff` in `on_new_pr`), finds a before path of `/dev/null`, and an after path that matches one of the metadata directory paths with a file extension that is not `.ini`.

### PR DESCRIPTION
**Pull request by g-k**
Fixes #43

### CODE REVIEW

**Comment by KiChjang** [comment]
Please claim the issue next time when you work on it. Other contributors may have also been looking into the issue and we would have duplicated efforts on the same issue.

**Comment by g-k** [comment]
Sorry!  I started looking into it and got carried away.  I'll be sure to claim it next time; people have worked on issues I had before and it sucked.

**Comment by jdm** [comment]
I apologize for forgetting about this; I'll review the changes tomorrow!

File: handlers/nonini_wpt_meta/__init__.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    TEST_DIRS_TO_CHECK = (
+        'wpt/metadata',
+        'wpt/mozilla/meta',
+    )
+
+    def on_pr_opened(self, api, payload):
+        diff = api.get_diff()
+        test_dirs_with_offending_files = set()
+
+        for line in diff.split('\n'):
```
Comment (jdm) [inline handlers/nonini_wpt_meta/__init__.py]: We should really be ignoring any lines that aren't of the form `diff --git` like the missing_test handler, or we could end up with false positives.

**Comment by jdm** [comment]
This looks really good! I think we can tighten up the check in order to avoid flagging things that aren't actually a problem, though.

**Comment by g-k** [comment]
Updated to only check `diff --git` lines and refactored a bit.

**Comment by jdm** [comment]
Thanks!

**Pull request merged**